### ISSUE TITLE
Keras Classifier Block

### ISSUE THREAD

**Issue by cstefanache** [ascentcore/research-ml-blocks#8]

**Comment by cstefanache** [comment]
Considering that the neural net will be compiled as follows:
`
self.model = keras.Sequential()
self.model.add(keras.layers.Dense(128, activation='sigmoid'))
self.model.add(keras.layers.Dense(64, activation='sigmoid'))
self.model.add(keras.layers.Dense(1, activation='sigmoid'))
`

publis the settings schema as such:
- Layers will be of type array and items will reference a sub schema
- The sub schema (Layers) will have the following properties: Unites and Activation Function
Layer size will be of type integer min: 1 max 1024
Activation Function will be of type `enum` and options are all known Keras Activation Functions
